spec/unit/iiif/v3/presentation/manifest_spec.rb

```diff
@@ -66,40 +71,139 @@ def initialize(hsh={})
     end
-    it 'raises IllegalValueError for default Sequence that is not written out' do
-      subject['id'] = manifest_id
-      subject.label = 'Book 1'
-      seq = IIIF::V3::Presentation::Sequence.new
-      seq['items'] = []
-      subject['items'] = [seq]
-      exp_err_msg = 'The default Sequence (the first entry of "items") must be written out in full within the Manifest file'
-      expect { subject.validate }.to raise_error(IIIF::V3::Presentation::IllegalValueError, exp_err_msg)
+    describe 'items' do
+      it 'raises MissingRequiredKeyError for items entry without values' do
+        subject['id'] = manifest_id
+        subject.label = 'Book 1'
+        subject['items'] = []
+        expect { subject.validate }.to raise_error(IIIF::V3::Presentation::MissingRequiredKeyError, seq_list_err)
+      end
+      it 'raises IllegalValueError for items entry that is not a Sequence' do
+        subject['id'] = manifest_id
+        subject.label = 'Book 1'
+        subject['items'] = [IIIF::V3::Presentation::Sequence.new, IIIF::V3::Presentation::Canvas.new]
+        expect { subject.validate }.to raise_error(IIIF::V3::Presentation::IllegalValueError, seq_entry_err)
+      end
+      describe 'raises IllegalValueError for default Sequence that is not written out' do
+        it 'Sequence has "items"' do
+          subject['id'] = manifest_id
+          subject.label = 'Book 1'
+          seq = IIIF::V3::Presentation::Sequence.new
+          seq['items'] = []
+          subject['items'] = [seq]
+          expect { subject.validate }.to raise_error(IIIF::V3::Presentation::IllegalValueError, def_seq_err)
+        end
+        # NOTE:  also allowing canvases as Universal Viewer currently only accepts canvases
+        #  see https://github.com/sul-dlss/osullivan/issues/27, sul-dlss/purl/issues/167
+        it 'Sequence has "canvases"' do
+          subject['id'] = manifest_id
+          subject.label = 'Book 1'
+          seq = IIIF::V3::Presentation::Sequence.new
+          seq['canvases'] = []
+          subject['items'] = [seq]
+          expect { subject.validate }.to raise_error(IIIF::V3::Presentation::IllegalValueError, def_seq_err)
+        end
+      end
+      it 'raises no error for Sequence with populated "items"' do
+        seq = IIIF::V3::Presentation::Sequence.new
+        seq['items'] = [IIIF::V3::Presentation::Canvas.new]
+        subject['items'] = [seq]
+        subject['id'] = manifest_id
+        subject.label = 'Book 1'
+        expect { subject.validate }.not_to raise_error
+      end
+      it 'raises no error for Sequence with populated "canvases"' do
+        seq = IIIF::V3::Presentation::Sequence.new
+        seq['canvases'] = [IIIF::V3::Presentation::Canvas.new]
+        subject['items'] = [seq]
+        subject['id'] = manifest_id
+        subject.label = 'Book 1'
+        expect { subject.validate }.not_to raise_error
```

Comment:
a single sequence without a label is allowed.  Labels are only required if there are multiple sequences.